libs/parallelism/algorithms/include/hpx/parallel/container_algorithms/replace.hpp

```diff
@@ -235,13 +250,18 @@ namespace hpx { namespace parallel { inline namespace v1 {
     ///           element in the destination range, one past the last element
     ///           copied.
     ///
+    // clang-format off
     template <typename ExPolicy, typename Rng, typename OutIter, typename T1,
         typename T2, typename Proj = util::projection_identity,
-        HPX_CONCEPT_REQUIRES_(hpx::is_execution_policy<ExPolicy>::value&&
-                hpx::traits::is_range<Rng>::value&& traits::is_projected_range<
-                    Proj, Rng>::value&& traits::is_indirect_callable<ExPolicy,
-                    std::equal_to<T1>, traits::projected_range<Proj, Rng>,
-                    traits::projected<Proj, T1 const*>>::value)>
+        HPX_CONCEPT_REQUIRES_(
+            hpx::is_execution_policy<ExPolicy>::value &&
+            hpx::traits::is_range<Rng>::value &&
+            traits::is_projected_range<Proj, Rng>::value &&
+            traits::is_indirect_callable<
+                ExPolicy,std::equal_to<T1>, traits::projected_range<Proj, Rng>,
+                    traits::projected<Proj, T1 const*>>::value
+        )>
+    // clang-format on
     typename util::detail::algorithm_result<ExPolicy,
         hpx::util::tagged_pair<
```

Comment:
I think for the other algorithms we have replaced the `tagged_pair`/`tagged_tuple` return types.

Reply:
Hm... I haven't even deprecated them. Strange.